libpay-stripe/src/test/scala/com/wix/pay/stripe/it/StripeGatewayIT.scala

```diff
@@ -72,6 +73,10 @@ class StripeGatewayIT extends SpecWithJUnit with Matchers {
       havePairs(pairs:_*) ^^ getRequestFields _
     }
 
+    def notContainsFields(fields: String*): Matcher[HttpRequest] = {
```

Comment:
Small suggestion: let us rename this method to `notContainFields` (singular) and rename `containsFields` to `containFields`- it will read properly in current usages.